exercises/queen-attack/queen_attack_test.py

```diff
@@ -1,92 +1,65 @@
 import unittest
 
-from queen_attack import board, can_attack
+from queen_attack import Queen
 
 
-# Tests adapted from `problem-specifications//canonical-data.json` @ v2.0.0
+# Tests adapted from `problem-specifications//canonical-data.json` @ v2.1.0
 
 class QueenAttackTest(unittest.TestCase):
 
+    # Test creation of Queens with valid and invalid positions
     def test_queen_valid_position(self):
         try:
-            board((1, 1), (2, 2))
+            Queen(2, 2)
         except ValueError:
             self.fail("Unexpected Exception")
 
     def test_queen_negative_row(self):
         with self.assertRaisesWithMessage(ValueError):
-            board((1, 1), (-2, 2))
+            Queen(-2, 2)
 
     def test_queen_invalid_row(self):
         with self.assertRaisesWithMessage(ValueError):
-            board((1, 1), (8, 4))
+            Queen(8, 4)
 
     def test_queen_negative_column(self):
         with self.assertRaisesWithMessage(ValueError):
-            board((1, 1), (2, -2))
+            Queen(2, -2)
 
     def test_queen_invalid_column(self):
         with self.assertRaisesWithMessage(ValueError):
-            board((1, 1), (4, 8))
+            Queen(4, 8)
 
+    # Test the ability of one queen to attack another
     def test_attack_false(self):
-        self.assertIs(can_attack((2, 4), (6, 6)), False)
+        self.assertIs(Queen(2, 4).can_attack(Queen(6, 6)), False)
 
     def test_attack_same_row(self):
-        self.assertIs(can_attack((2, 4), (2, 6)), True)
+        self.assertIs(Queen(2, 4).can_attack(Queen(2, 6)), True)
 
     def test_attack_same_column(self):
-        self.assertIs(can_attack((4, 5), (2, 5)), True)
+        self.assertIs(Queen(4, 5).can_attack(Queen(2, 5)), True)
 
     def test_attack_diagonal1(self):
-        self.assertIs(can_attack((2, 2), (0, 4)), True)
+        self.assertIs(Queen(2, 2).can_attack(Queen(0, 4)), True)
 
     def test_attack_diagonal2(self):
-        self.assertIs(can_attack((2, 2), (3, 1)), True)
+        self.assertIs(Queen(2, 2).can_attack(Queen(3, 1)), True)
 
     def test_attack_diagonal3(self):
-        self.assertIs(can_attack((2, 2), (1, 1)), True)
+        self.assertIs(Queen(2, 2).can_attack(Queen(1, 1)), True)
 
     def test_attack_diagonal4(self):
-        self.assertIs(can_attack((2, 2), (5, 5)), True)
+        self.assertIs(Queen(2, 2).can_attack(Queen(5, 5)), True)
 
-    # Tests beyond this point are not part of the canonical data.
-
-    # If either board or can_attack are called with an invalid board position
-    # they should raise a ValueError with a meaningful error message.
+    # Track-specific tests
     def test_invalid_position_can_attack(self):
```

Comment:
This is redundant; since there are already tests to confirm that `Queen()` validates the individual position, this is implicitly tested.

Reply:
Good catch!